llvm/include/llvm/Support/PropertySetIO.h

```diff
@@ -187,6 +187,7 @@ class PropertySetRegistry {
       "SYCL/composite specialization constants";
   static constexpr char SYCL_DEVICELIB_REQ_MASK[] = "SYCL/devicelib req mask";
   static constexpr char SYCL_KERNEL_PARAM_OPT_INFO[] = "SYCL/kernel param opt";
+  static constexpr char SYCL_IS_ESIMD_IMAGE[] = "SYCL/is ESIMD image";
```

Comment:
@DenisBakhvalov, as we agreed, can you please change the category name for this to something like SYCL/misc or SYCL/binary image?